addons/medical/functions/fnc_deserializeState.sqf

```diff
@@ -0,0 +1,98 @@
+#include "script_component.hpp"
+/*
+ * Author: BaerMitUmlaut
+ * Deserializes the medical state of a unit and applies it.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ * 1: State as JSON <STRING>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [player, _json] call ace_medical_fnc_deserializeState
+ *
+ * Public: Yes
+ */
+params [["_unit", objNull, [objNull]], ["_json", "{}", [""]]];
+
+if (isNull _unit) exitWith {};
+if (!local _unit) exitWith { ERROR_1("unit [%1] is not local",_unit) };
+
+// If unit is not initialized yet, wait until event is raised
+if !(_unit getVariable [QGVAR(initialized), false]) exitWith {
+    [QGVAR(initialized), {
```

Comment:
Shouldn't this event name be `QEGVAR(medical_status,initialized)`?

Reply:
@BaerMitUmlaut ?

Reply:
Fixed.